environment/container/kubernetes/k3s.go

```diff
@@ -156,18 +157,24 @@ func installK3sCluster(
 		"--write-kubeconfig-mode", "644",
 	}, k3sArgs...)
 
-	// replace ip address if networking is enabled
-	ipAddress := limautil.IPAddress(config.CurrentProfile().ID)
-	if ipAddress == "127.0.0.1" {
-		args = append(args, "--flannel-iface", "eth0")
-	} else {
-		if !hasK3sArg(k3sArgs, "--advertise-address") {
-			args = append(args, "--advertise-address", ipAddress)
+	a.Retry("waiting for VM IP address", time.Second*2, 10, func(retryCount int) error {
```

Comment:
why not simply retry 3 times at 5 seconds interval?

```suggestion
	a.Retry("waiting for VM IP address", time.Second*5, 4, func(retryCount int) error {
```

Reply:
Yeah that's fine by me. I just arbitrarily picked 2 seconds because I figured most of the time the IP should be assigned very soon after this check, but that's totally just a guess.